biran.gemspec

```diff
@@ -13,7 +13,7 @@ Gem::Specification.new do |s|
   s.summary     = "Helper for generating config generate tasks."
   s.description = "Biran is the guy that will help you generate config files for your rail app."
   s.license     = "MIT"
-  s.required_ruby_version = ">= 2.6.0"
+  s.required_ruby_version = ">= 2.7.0"
```

Comment:
With this change, we will likely have the next release be `0.2.0` to signify the change